configure.ac

```diff
@@ -141,6 +160,7 @@ Configure summary:
         WARN_CFLAGS:            ${WARN_CFLAGS}
         C++ Compiler:           ${CXX}
         CXXFLAGS:               ${CXXFLAGS}
+        Elogind support:        ${use_elogind}
```

Comment:
This should be moved down next to the line for systemd, shouldn't it?

Reply:
I only was following the convention of sorting alphabetically :-)